metadata.json

```diff
@@ -35,6 +35,10 @@
     {
       "name": "katello/certs",
       "version_requirement": ">= 6.0.0 < 8.0.0"
+    },
+    {
+      "name": "theforeman/pulpcore",
```

Comment:
I noticed this too after the release and we should have done this before already.